<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>716</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>716</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/Main Icon.png</normaloff>:/images/Main Icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ratNumberLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Крыса №:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ratNumberLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ratDescLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ratDescLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание: </string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>701</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="LF1RadioButton">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="LF2RadioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="LF3RadioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="HF1RadioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="HF2RadioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="HF3RadioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="LF1Button">
         <property name="text">
          <string>НЧ1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LF2Button">
         <property name="text">
          <string>НЧ2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LF3Button">
         <property name="text">
          <string>НЧ3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="HF1Button">
         <property name="text">
          <string>ВЧ1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="HF2Button">
         <property name="text">
          <string>ВЧ2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="HF3Button">
         <property name="text">
          <string>ВЧ3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ECGChartView" name="graphicsView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="zoomInButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="zoomOutButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scrollRightButton">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scrollLeftButton">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scrollUpButton">
         <property name="text">
          <string>^</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scrollDawnButton">
         <property name="text">
          <string>v</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scaleInButton">
         <property name="text">
          <string>M+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scaleOutButton">
         <property name="text">
          <string>M-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="ratRILabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>НИИ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ratRILineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="waveformButton">
       <property name="text">
        <string>Осциллограмма</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="waveletButton">
       <property name="text">
        <string>Вейвлет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="histogrammButton">
       <property name="text">
        <string>Гистограмма</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spectrumButton">
       <property name="text">
        <string>Спектр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Запись</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="add_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Получить данные</string>
    </property>
    <addaction name="ZEtLab_action"/>
    <addaction name="getAll_action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="settings_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="open_action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="add_action">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="ZEtLab_action">
   <property name="text">
    <string>Получить файы ZetLab</string>
   </property>
  </action>
  <action name="getAll_action">
   <property name="text">
    <string>Получить копию всех файлов</string>
   </property>
  </action>
  <action name="settings_action">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ECGChartView</class>
   <extends>QGraphicsView</extends>
   <header>ecgchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
